tests/conftest.py

```diff
@@ -158,3 +163,28 @@ def override_get_qc_db():
     client = TestClient(app)
 
     return client
+
+
+def insert_from_yaml(session, dir_path, module_name):
+
+    # Load the schema module where the table ORM classes are defined.
+    module = importlib.import_module(module_name)
+
+    # Find all files in a given directory.
+    dir_obj = pathlib.Path(dir_path)
+    file_paths = list(str(f) for f in dir_obj.iterdir())
+    file_paths.sort()
+
+    for file_path in file_paths:
+        with open(file_path, "r") as f:
+            (head, file_name) = os.path.split(file_path)
+            # File name example: 200-PacBioRun.yml
+            m = re.match(r"\A\d+-([a-zA-Z]+)\.yml\Z", file_name)
+            if m is None:
```

Comment:
Not a fan of silently ignoring erroneous test data files. It'll error anyway if it's not yaml, and you're only guarding against test writers.

Reply:
Changed to an error